install/installer/setup_ssh_key

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This has to be an empty file that needs to exist and will be updated during installation